spec/shop_spec.rb

```diff
@@ -23,5 +23,12 @@
       #rspec matcher tests if object passed to expect is = to object passed to eq
     end
   end
-end
 
+  describe "#all" do
+    #test db number of shops = Shop.all.count
+    it 'should have same number of shops' do
+      shop = Shop.new "cafe"
```

Comment:
line 30 is a bit redundant here. Since you are not doing anything with `shop` object, you are practically introducing more noise to your code.

The question is then. What is your number 55 based on?
A valid test in this case would be, that you first pull the number of rows in the `shops` table from the database and assign it to a variable.

Then you perform an equality assertion based on that variable against the `Shop.all.count`

Reply:
Hi Alex, do you mean assign the variable within the test? So rows = DB[:shops].all this would use the same piece of code as the class method, is this what we are trying to eq? 

Shop.all.count returns an integer and rows returns an array, so is it best to make them both arrays or both integers?

Reply:
`Hi Alex, do you mean assign the variable within the test? ` => yes

I think i was a bit confused about your magic number 55.... therefore i made those comments yesterday.

Actually, your current setup of 
`rows = DB[:shops].all` and 
` expect(Shop.all).to eq(rows)`
would suffice already 👍